buildtest/cli/__init__.py

```diff
@@ -396,7 +398,29 @@ def misc_menu(subparsers):
         ],
         help="Show help message for command",
     )
-
+def profile_menu(subparsers):
+    profile_parser = subparsers.add_parser(
+        "help",
+        config=["-h"],
+        compliers="config profile list",
+    )
+    profile_parser.add_arguments(
+        "command",
+        choices=[
+            "compilers",
+            "co",
+            "edit",
+            "e",
+            "executors",
+            "ex",
+            "path",
+            "p",
+            "systems",
+            "validate",
+            "view",
+            "-v"
+        ],
+    )
```

Comment:
This does not make any sense. You should expect to see a command name `buildtest config profiles` and this is not doing what you want. If you dont see this command then you have not done this correctly.